src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -247,9 +247,30 @@ <h3>Headers</h3>
                     <!-- /ko -->
                     <!-- ko if: !header.options || header.options.length === 0 -->
                     <div class="input-group">
+                        <!-- ko if: !header.secret -->
                         <input type="text" autocomplete="off" class="form-control form-control-sm" placeholder="value"
                             spellcheck="false" aria-label="Header value"
                             data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
+                        <!-- /ko -->
+                        <!-- ko if: header.secret -->
+                        <div style="width: 100%;">
+                            <!-- ko if: !header.revealed() -->
+                            <input type="password" autocomplete="off" class="form-control form-control-sm password-input"
+                                placeholder="value" spellcheck="false" aria-label="Header value"
+                                data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
+                            <!-- /ko -->
+                            <!-- ko if: header.revealed() -->
+                            <input type="text" autocomplete="off" class="form-control form-control-sm password-input"
+                                placeholder="value" spellcheck="false" aria-label="Header value"
+                                data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
```

Comment:
`event: { keyup: $component.updateRequestSummary }` - instead `keyup` event you could [subscribe to changes](https://knockoutjs.com/documentation/observables.html#explicitly-subscribing-to-observables) of the observable: `myObservable.subscribe(() => { ... }`. That's because the field value may change not only on `keyup` - user can copy paste values with mouse or another script can update the value (for instance, we're setting up an OAuth token into `Authorization` header when user authenticates in Test console.